item-api/src/main/java/com/example/auction/item/api/PaginatedSequence.java

```diff
@@ -6,6 +6,7 @@
 /**
  * A partial sequence of elements, with metadata for retrieving additional pages.
  */
+// TODO: move to a common sbt project to reuse from searchApi
```

Comment:
Yes. There's some bits and pieces that could be shared amongst projects. Pagination and how it's modelled on the service end, testing tools (`Await`, `expectRethrows`,...).